step.yml

```diff
@@ -225,8 +225,12 @@ inputs:
       is_sensitive: true
   - generate_universal_apk_if_none: "true"
     opts:
-      title: "Force generate universal APK for AABs"
+      title: "Force generate universal APK for AABs (DEPRECATED)"
       description: |
+        Universal APK generation is going to be  **DEPRECATED** with the next
+        major release! Please use the _Export Universal APK_ Step instead! You 
+        may find further information about the Step [here](https://devcenter.bitrise.io/deploy/android-deploy/exporting-a-universal-apk-from-an-aab/).
```

Comment:
may find further information about the Step [here](https://devcenter.bitrise.io/deploy/android-deploy/exporting-a-universal-apk-from-an-aab/).  >> .... may find further information about the Step in our [Exporting a universal APK from an AAB](https://devcenter.bitrise.io/deploy/android-deploy/exporting-a-universal-apk-from-an-aab/) article.